docs/reference/authentication.md

```diff
@@ -27,6 +27,17 @@ Web UI at Administration / Users to create users.  Users have an access key
 `AKIA...` and an associated secret access key.  These credentials are valid
 to log into the Web UI, or to authenticate programmatic requests to the API
 Server or the S3 Gateway.
+git
```

Comment:
I don't understand this line.

Reply:
My mistake. (Probably just tried to type `git` in the command line and accidentally typed it into the doc). Fixed